Makefile

```diff
@@ -70,7 +68,7 @@ ifeq (, $(shell which controller-gen))
 	CONTROLLER_GEN_TMP_DIR=$$(mktemp -d) ;\
 	cd $$CONTROLLER_GEN_TMP_DIR ;\
 	go mod init tmp ;\
-	go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.4.0 ;\
+	go get sigs.k8s.io/controller-tools/cmd/controller-gen@latest ;\
```

Comment:
Do you think we should pin this to a specific hash?

Reply:
I've pinned this with this commit https://github.com/monzo/egress-operator/pull/44/commits/b8a2e9c55ff7b078396298191b05d493c33abcb1